### ISSUE TITLE
When using an IME, escape will close editing a new email

### ISSUE THREAD

**Issue by zakkudo** [tutao/tutanota#5891]
**Describe the bug**
When I use japanese to enter text in tutanota, if I press escape to cancel my input in the IME tutanota cancels the whole email.

**To Reproduce**
1. Create a new email
2. Switch to japanese
3. type some letters like 'ha'
4. Notice it is underlined.  Press escape, expecting it to cancel your input and it closes the whole message.

**Expected behavior**
It should cancel the staged input text without closing the email.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 3.118.7 of tutanota

**Additional context**
tutanota is likely using the wrong event listener for escape which is causing this problem.  This is a bad enough issue that it will likely scare off any users who have to use a foreign language that needs an IME. (Japanese, Chinese, etc)

**Comment by ganthern** [comment]
That's annoying. Can you give an example of an IME where this happens? What's your setup? I assume it's also a problem in the web app?

**Comment by zakkudo** [comment]
What happens when you press escape to stop the IME input.

https://github.com/tutao/tutanota/assets/1668500/8a94c3b2-a5a5-4d3b-82f4-0f9d1335e9ee

**Comment by zakkudo** [comment]
What should happen when you hit the escape button is it should remove the text.  The IME should be the only thing receiving the key press.

https://github.com/tutao/tutanota/assets/1668500/8fe9b3b1-60ac-406c-a8cf-c7ccdc9b96ae

**Comment by ganthern** [comment]
thank you so much, that makes it much clearer!
